config.js

```diff
@@ -1441,6 +1441,23 @@ var config = {
     // dialInConfCodeUrl is the conference mapper converting a meeting id to a PIN used for dial-in
     // or the other way around (more info in resources/cloud-api.swagger)
 
+    // You can use external service for authentication that will redirect back passing a jwt token
+    // You can use tokenAuthUrl config to point to a URL of such service.
+    // The URL for the service supports few params which will be filled in by the code.
+    // tokenAuthUrl:
```

Comment:
Note for discussion later: how about reorganizing with a section?

```
tokeAuth: {
    loginUrl: 'https://blah',
    logoutUrl: 'https://bleh',
    autoRedirect: true,
    ...
}
```

Reply:
Make sense. Bit let's do the deprecating and all that is needed in a future PR when we don't have many moving pieces.

Reply:
Sure, that's why I mentioned for later :-)